README.md

```diff
@@ -1,3 +1,41 @@
-#Hollow API Generator Nebula Plugin
+=======
+# Hollow API Generator Nebula Plugin
 
-This plugin will provide a task for generating a Hollow consumer API, as described [here](http://hollow.how/getting-started/#consumer-api-generation).
+This plugin provides task for generating hollow consumer api, that is 
+described [here](http://hollow.how/getting-started/#consumer-api-generation)
+
+## Usage
+In order to use, add plugin dependency to your buildscript
+```
+buildscript {
+    repositories {
+        mavenLocal()
```

Comment:
I see your `TODO` further down regarding this. I believe there a TODO is fine. For this README we can write instructions assuming hollow plugin has been published.